src/client/clientmedia.cpp

```diff
@@ -168,6 +168,15 @@ void ClientMediaDownloader::step(Client *client)
 	}
 }
 
+std::string ClientMediaDownloader::makeReferer(Client *client)
```

Comment:
if you are searching for IPv6 we may do a less naive string find to validate inputs. Or we should refacto a bit to gather a properly serialized address to be used elsewhere from our Address objects

Reply:
This isn't about validation. We just have a string that's either a valid domain, IPv4 or IPv6 and we want to safely encode it for an URL.